apps/assisted_tagging/mutation.py

```diff
@@ -39,6 +41,16 @@
     serializer_class=MissingPredictionReviewGqlSerializer,
 )
 
+TriggerDraftEntryInputType = generate_input_type_for_serializer(
+    "TriggerDraftEntryInputType",
+    serializer_class=TriggeredDraftEntryGqlSerializer
+)
```

Comment:
Let's use consistent naming
```suggestion
TriggerAutoDraftEntryInputType = generate_input_type_for_serializer(
    "TriggerAutoDraftEntryInputType",
    serializer_class=TriggeredAutoDraftEntryGqlSerializer
)
```